auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -335,11 +390,79 @@ export default defineComponent({
       clearAllFilters()
     }
 
+    const isCurrentOrganization = (orgId: number) => {
+      return orgId === store.state.org.currentOrganization.id
+    }
+
+    const actionHandler = (business: Business) => {
+      const aii = business.affiliationInvites[0]
+      const status1 = aii.status
```

Comment:
status1, seems weird to me - what does the 1 signify? 

why not const status (might be a conflict)

or const inviteStatus?

Reply:
ok, will try to not have those (naming from my IDE) I usually rename them, but here they were just very local so I ignored it.
Updated it now as, like you suggested, it is better.